regolith/main.py

```diff
@@ -169,6 +169,14 @@ def create_parser():
              "the build will be for only those people",
         default=None,
     )
+    bldp.add_argument(
```

Comment:
I think this is a bit too specific to add as a command line arg to the regolith project as a whole.  Can we work around it?
e.g., we could use ``--since`` and only build beamplans more recent than since, or all plans otherwise, sthg like that?

Reply:
Currently, by default, if user doesn't use the argument beamtime, the plans for all beamtime will be reported. Each beamtime generate a file.

I use the beamtime because beamtime is a easy way to filter the docs. I don't want to do too complicated filter like the time because in the future, we will refractor all the code to use mongo query to query data and we don't need to spend too much time to write filter by  our own.

So how about delete this beamtime argument and just let the builder build all beamtime plan files?